website/src/components/SearchPage/Table.tsx

```diff
@@ -95,6 +100,14 @@ export const Table: FC<TableProps> = ({
         }
     };
 
+    const handleRowSelectToggle = (e: React.MouseEvent<HTMLTableCellElement>, seqId: string) => {
+        if (selectedSeqs.includes(seqId)) {
+            setSelectedSeqs(selectedSeqs.filter((item) => item !== seqId));
```

Comment:
```suggestion
            setSelectedSeqs((prevSelectedSeqs) => prevSelectedSeqs.filter((item) => item !== seqId));
```

This isn't necessarily essential here - I'm not sure But since you mentioned you were new to react I thought I'd mention this pattern, which I tend to like for things like this, which ensures that the operation happens on the current up to date version of the state. Otherwise you can sometimes get weird behaviour where e.g. if you unclicked two boxes very fast the second box click might restore the first box because its `selectedSeqs` still includes that first box.

Reply:
(same below)

Reply:
thanks, very good suggestion! I've fixed both instances here: https://github.com/loculus-project/loculus/pull/2917/commits/cf9ab14a549ae09efda5a4fb8712352b1fbf8900